memorious/operations/documentcloud.py

```diff
@@ -32,24 +34,25 @@
 
 
 def documentcloud_query(context, data):
-    host = context.get("host", DEFAULT_HOST)
+    host = context.get("host", API_HOST)
     instance = context.get("instance", DEFAULT_INSTANCE)
     query = context.get("query")
     page = data.get("page", 1)
 
-    search_url = urljoin(host, "search/documents.json")
+    search_url = urljoin(host, "/api/documents/search")
     res = context.http.get(
         search_url, params={"q": query, "per_page": 100, "page": page}
     )
-    documents = res.json.get("documents", [])
+
+    documents = res.json.get("results", [])
     for document in documents:
```

Comment:
I think we should check if the document is a pdf or not by looking at the `original_extension` field just to be on the safe side.

Reply:
I gave this a thumbs up without fully thinking it through. I'm interested to understand what to do do in an instance where the original_extension is not a pdf?

From the brief look I've had it seems as though documents are converted to pdf at some stage in the documentcloud process. Take this result:

`{
      "id": "764010",
      "user": 1569,
      "organization": {
        "id": 56,
        "avatar_url": "https://cdn.muckrock.com/static/images/avatars/organization.png",
        "individual": false,
        "name": "Texas Tribune",
        "slug": "texastribune",
        "uuid": "6e7d35b2-af22-430b-bc7c-d0676dafe32f"
      },
      "access": "public",
      "status": "success",
      "title": "13_MA20801_10AA_Money Money Money",
      "slug": "13_ma20801_10aa_money-money-money",
      "language": "eng",
      "created_at": "2013-08-21T03:40:47.144Z",
      "updated_at": "2020-11-12T18:45:49.157Z",
      "page_count": 2,
      "projects": [
        10186,
        10255
      ],
      "original_extension": "docx",
      "file_hash": "7fc3690bfbaf4cd785ce6bab20c3394784eac0bf",
      "highlights": {
        "page_no_1": [
          "Algebra 2\nHS Mathematics\nUnit: 08 Lesson: 01\n\n<em>Money</em>, <em>Money</em>, <em>Money</em>\n\n<em>Money</em> Owed ($)\n\nCollege Loan. "
        ]
      },
      "data": {},
      "asset_url": "https://assets.documentcloud.org/",
      "edit_access": false
    }`

The original extension here is a docx, however the two generated urls both work with assuming the doc is a PDF:

https://assets.documentcloud.org/documents/764010/13_ma20801_10aa_money-money-money.pdf
https://www.documentcloud.org/documents/764010-13_ma20801_10aa_money-money-money

Do you think that the mime_type should be updated to reflect the original document type?

Reply:
Yes, you're right. Documents are being converted to PDF.

They do say "We currently only support PDF documents." in the API docs. So I think we can safely assume all documents we get in the result are going to be PDFs in the end.

Reply:
@sunu Agreed, it's not a difficult change in the future if they start supporting other document formats.